eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -96,12 +96,16 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation) {
     # If the package is of a dev version there may be language-specific needs to 
     # specify the appropriate version. For example, in the case of JS, the dev 
     # version is always 'dev' when interacting with NPM.
-    if ($GetDocsMsLanguageSpecificPackageInfo -and (Test-Path "Function:$GetDocsMsLanguageSpecificPackageInfo")) { 
-      $packageInfo = &$GetDocsMsLanguageSpecificPackageInfo $packageInfo
+    if ($GetDocsMsDevLanguageSpecificPackageInfoFn -and (Test-Path "Function:$GetDocsMsDevLanguageSpecificPackageInfoFn")) { 
+      $packageInfo = &$GetDocsMsDevLanguageSpecificPackageInfoFn $packageInfo
+    } else {
+      # Default: use the dev version from package info as the version for
+      # downstream processes
+      $packageInfo.Version = $packageInfo.DevVersion
     }
   }
 
-  $packageMetadataArray = (Get-CSVMetadata).Where({ $_.Package -eq $packageInfo.Name })
+  $packageMetadataArray = (Get-CSVMetadata).Where({ $_.Package -eq $packageInfo.Name -and $_.GroupId -eq $packageInfo.Group -and $_.Hide -ne 'true' -and $_.New -eq 'true' })
```

Comment:
Not sure if this is going to come into play here yet but if we ever enable strict mode .GroupId will fail because it only exists on some language objects.